department-of-veterans-affairs/va.gov-cms
# Vagov 3600 pension migration

To test:
1. go to `/admin/structure/migrate/manage/va_new_benefits/migrations`
2. Execute 'Migrate Pension benefits pages from VA.gov', 'Migrate Pension sidebar menu', and 'Pension landing page - NEVER ROLLBACK' (check 'Update' for landing page)
3. Go to `/admin/content/benefits?title=&moderation_state=editorial-draft&type_1=All`. You should see 8 Benefits detail pages and 1 Benefits hub landing page with owner, 'Pension benefits hub'.
